### PR TITLE
Update CHANGELOG for v6.2.0

### PR DESCRIPTION
**Pull request by byucesoy**
CHANGELOG changes for 6.2 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+
+* Enables router planner for queries on range partitioned table
+
+* Changes `TRUNCATE` to only erase local data if enable_ddl_propagation is off
+
+* Starts to execute DDL on coordinator before workers
+
+* Fixes a bug causing incorrectly reading invalidated cache
+
+* Fixes a bug related to creation schemas of in workers with incorrect owner
```
Comment (lfittl) [inline CHANGELOG.md]: `relate to creation of schemas in workers`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+
+* Fixes a bug related to `EXPLAIN ANALYZE` with DML queries
+
+* Fixes a bug related to SQL functions in FROM clause
+
+* Adds a GUC variable to report cross shard queries
+
+* Fixes a bug related to partition columns without native hash function
+
+* Adds new internal infastructures and tests to increase development process
```
Comment (lfittl) [inline CHANGELOG.md]: `new internal infrastructure and tests to improve development process`

**Comment by lfittl** [comment]
@byucesoy Just noticed a few typos as I was looking this over - please don't take this as an actual review - I'll leave that to @jasonmp85 :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,63 @@
...
+
+* Improves planner performance for joins involving co-located tables
+
+* Improves shard copy performance by creating indexes after copy
+
+* Improves task-tracker performance by batching several status checks
+
+* Enables router planner for queries on range partitioned table
+
+* Changes `TRUNCATE` to only erase local data if enable_ddl_propagation is off
```
Comment (jasonmp85) [inline CHANGELOG.md]: `enable_ddl_propagation` should be backticked, if we have room.

**Comment by jasonmp85** [comment]
Eh, it's a `CHANGELOG`, the copyediting is all there is :-P. This looks pretty good to me. Merging now.

**Comment by jasonmp85** [comment]
General comment: _adds **new**__ is redundant, since if we're adding something it can be assumed to be new.

**Pull request merged**